### ISSUE TITLE
npm install error

### ISSUE THREAD

**Issue by traschke** [AndyMeps/ng2-color-picker#2]
Hi,

I am unable to install the package via npm. It outputs the following error:
```
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "ng2-color-picker" "--save"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code EREADFILE

npm ERR! Error extracting C:\Users\Timo\AppData\Roaming\npm-cache\ng2-bootstrap\1.1.15\package.tgz archive: ENOENT: no such file or directory, open 'C:\Users\Timo\AppData\Roaming\npm-cache\ng2-bootstrap\1.1.15\package.tgz'
```
btw I'm on Windows 10.

I took a look at the npm-cache folder and discovered that npm downloaded version 1.1.16 of ng2-bootstrap.

Could it be that there is some version targeting wrong in the package.json of this package?

**Comment by AndyMeps** [comment]
Hi traschke,

It seems there is something wrong with that version of ng2-bootstrap suddenly.. I'm migrating the library to support the latest version of ng2-bootstrap and angular 2 as a result. This should resolve this issue. I'll post the release version to use shortly, however it will require you are using the latest version of angular 2 libraries.

Regards,

Andy

**Comment by AndyMeps** [comment]
Hi, this should be fixed with v1.1.0 of ng2-color-picker. Please let me know if you have further issues; will leave this open for 7 days for feedback.

Kind regards,

Andy
